### PR TITLE
Bug fix : unset the TCCL on wrong thread when executing blocking handler

### PR DESCRIPTION
**Pull request by vietj**
@purplefox please review

### CODE REVIEW

File: src/main/java/io/vertx/core/impl/ContextImpl.java [hunk 1]
```diff
@@ -228,10 +232,6 @@ public EventLoop getEventLoop() {
       });
     } catch (RejectedExecutionException ignore) {
       // Pool is already shut down
-    } finally {
-      if (blockingCodeHandler != null) {
```
Comment (purplefox) [inline src/main/java/io/vertx/core/impl/ContextImpl.java]: agreed that this block should be removed

File: src/main/java/io/vertx/core/impl/ContextImpl.java [hunk 2]
```diff
@@ -221,6 +221,10 @@ public EventLoop getEventLoop() {
           }
         } catch (Throwable e) {
           res.fail(e);
+        } finally {
+          if (blockingCodeHandler != null) {
```
Comment (purplefox) [inline src/main/java/io/vertx/core/impl/ContextImpl.java]: I'm thinking that this block is actually not necessary - we don't explicit unset context in other cases, only at close so I think that should be fine in this case too (?)

Comment (vietj) [inline src/main/java/io/vertx/core/impl/ContextImpl.java]: ok

File: src/main/java/io/vertx/core/impl/ContextImpl.java (hunk 1 above)
Comment (vietj) [inline src/main/java/io/vertx/core/impl/ContextImpl.java]: that was leaving null thread contextclassloader of the junit thread and issues in subsequents jvm-npm.js deployment

**Pull request merged**